ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -395,8 +398,14 @@ objects:
         sed -i '/^  channel:/d' /tmp/artifacts/installer/manifests/cvo-overrides.yaml
 
         if [[ "${KURYR_ENABLED:-false}" != "true" ]]; then
-          echo "Forcing OpenShiftSDN by modifying manifests"
-          sed -i -e 's/networkType: .*$/networkType: OpenShiftSDN/g' /tmp/artifacts/installer/manifests/cluster-network-02-config.yml
+          if [[ "${CLUSTER_VARIANT:-sdn}" == "sdn" ]]; then
+            echo "Forcing OpenShiftSDN by modifying manifests"
+            sed -i -e 's/networkType: .*$/networkType: OpenShiftSDN/g' /tmp/artifacts/installer/manifests/cluster-network-02-config.yml
+          fi
+          if [[ "${CLUSTER_VARIANT:-sdn}" == "ovn" ]]; then
+            echo "Forcing OVNKubernetes by modifying manifests"
+            sed -i -e 's/networkType: .*$/networkType: OVNKubernetes/g' /tmp/artifacts/installer/manifests/cluster-network-02-config.yml
```

Comment:
You don't need that, just set whatever SDN you want in install-config.yaml.

Reply:
this is not implemented in the yaml file